pulser/_seq_drawer.py

```diff
@@ -199,6 +202,44 @@ def phase_str(phi):
         a.set_ylabel(r'$\Omega$ (rad/µs)', fontsize=14, labelpad=10)
         b.set_ylabel(r'$\delta$ (rad/µs)', fontsize=14)
 
+        if draw_phase_area:
+            top = False  # Variable to track position of box, top or center.
+            draw_phase = any(
+                seq_.type.phase != 0 for seq_ in seq._schedule[ch]
+                if isinstance(seq_.type, Pulse)
+            )
+            for pulse_num, seq_ in enumerate(seq._schedule[ch]):
+                # Select only `Pulse` objects
+                if isinstance(seq_.type, Pulse):
+                    if sampling_rate:
+                        area_val = np.sum(cs_amp(
+                            np.arange(seq_.ti, seq_.tf) * 1e-3
+                        )) * 1e-3 / np.pi
+                    else:
+                        area_val = seq_.type.amplitude.integral / np.pi
+                    phase_val = seq_.type.phase / np.pi
+                    x_plot = (seq_.ti + seq_.tf) / 2 / time_scale
+                    if (
+                        seq._schedule[ch][pulse_num-1].type == "target"
+                        or not top
+                    ):
+                        y_plot = np.max(seq_.type.amplitude.samples) / 2
+                        top = True  # Next box at the top.
+                    elif top:
+                        y_plot = np.max(seq_.type.amplitude.samples)
+                        top = False  # Next box at the center.
+                    area_fmt = (r"A: $\pi$" if area_val == 1
```

Comment:
You should check for the rounded value, ie:
```suggestion
                    area_fmt = (r"A: $\pi$" if round(area_val, 2) == 1
```

Reply:
Thanks, I used your suggestion.